common/test.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <test.h>
+
+#if TEST
+
+void
+run_tests(void){
+	test_crust_boots();
+}
+
+int
+test_crust_boots()
+{
+	test("Boot: PASS");
+	return 1;
```

Comment:
There's no need for this to return a value. The success/fail message is already printed (and you're not using the returned value).

Reply:
done